sorting/counting_sort_2.c

```diff
@@ -0,0 +1,33 @@
+#include<stdio.h>
+ 
+void counting_sort(int a[],int n,int max)

```

Comment:
Functions should be documented explaining the use of its parameters, a brief description, and what it returns.